src/browser/components/atoms/form-checkbox/index.js

```diff
@@ -1,24 +1,35 @@
 import React,{Component} from 'react'
 
 export default class FormCheckbox extends Component {
-  constructor() {
+  constructor(props) {
     super()
+    this.state = {}
+    props.options.forEach( option => {
+      this.state[option] = false
+    })
   }
 
+  transformData(){
```

Comment:
can you call this function `selectChecked`? `transformData` is too generic and doesn't tell me what is happening